packages/visx-xychart/src/components/Tooltip.tsx

```diff
@@ -275,34 +302,9 @@ function TooltipInner<Datum extends object>({
               </svg>
             </TooltipInPortal>
           )}
-          {glyphProps.map(({ left, top, fill, stroke, strokeWidth, radius }, i) =>
-            top == null || left == null ? null : (
-              <TooltipInPortal
-                key={i}
-                className="visx-tooltip-glyph"
-                left={left}
-                top={top}
-                offsetLeft={0}
-                offsetTop={0}
-                detectBounds={false}
-                style={TOOLTIP_NO_STYLE}
-              >
-                <svg width={(radius + strokeWidth) * 2} height={(radius + strokeWidth) * 2}>
-                  {/** @TODO expand to support any @visx/glyph glyph */}
-                  <circle
-                    cx={radius + strokeWidth}
-                    cy={radius + strokeWidth}
-                    r={radius}
-                    fill={fill}
-                    stroke={stroke}
-                    strokeWidth={strokeWidth}
-                    paintOrder="fill"
-                    {...glyphStyle}
-                  />
-                </svg>
-              </TooltipInPortal>
-            ),
-          )}
+          {glyphs.map((glyph, i) => (
+            <React.Fragment key={i}>{glyph}</React.Fragment>
```

Comment:
any reason to not include the `TooltipInPortal`? I think this means the glyph(s) would be rendered inside the svg container and the rest of the tooltip elements would be in portals 🤔

Reply:
Because it needs to know the dimensions of the glyph in order to position itself correctly

Reply:
@williaster I think we can either do a full width/height portal
```tsx

          {glyphs.map((glyph, i) => (
            <TooltipInPortal
              key={i}
              className="visx-tooltip-glyph"
              left={0}
              top={0}
              offsetLeft={0}
              offsetTop={0}
              detectBounds={false}
              style={TOOLTIP_NO_STYLE}
            >
              <svg width={width} height={height}>
                {glyph}
              </svg>
            </TooltipInPortal>
          ))}
```

or just

```
          <svg>
            {glyphs.map((glyph, i) => (
              <React.Fragment key={i}>{glyph}</React.Fragment>
            ))}
          </svg>
```

Reply:
makes sense. I agree the crosshairs could potentially be moved out of the portals as well as it's not as important if they get clipped (that was the main impetus for rendering the tooltip in a portal).

Reply:
Actually, I can make it work using `overflow` on the `svg`. Ironically I was the one who also came up with [that solution](https://github.com/airbnb/visx/issues/990#issue-779509740) for the crosshairs 🤦.